Orion.Net.Scripts.Common/Diagnostics/SendImageContentClientScript.cs

```diff
@@ -10,45 +10,44 @@ namespace Orion.Net.Scripts.Common.Diagnostics
     /// <summary>
     /// Take a screenshot on the client computer
     /// </summary>
-    public class TakeScreenShotClientScript : BaseClientScript
+    public class SendImageContentClientScript : BaseClientScript
     {
-        public TakeScreenShotClientScript(Connector connector)
+        public SendImageContentClientScript(Connector connector)
             : base(connector)
         {
             identifier = Guid.NewGuid();
             AvailableParameters.Add(new ScriptParameter()
             {
-                Name = "filePath"
+                Name = "sendImage"
             });
             AvailableParameters.Add(new ScriptParameter()
             {
                 Name = "args"
             });
         }
 
-        public override string Title => "Take screenshot";
+        public override string Title => "Send Image";
 
         private readonly Guid identifier;
         public override Guid Identifier => identifier;
 
         public override async Task Execute(string parameters)
         {
             var paramItems = parameters.ExtractParams();
-            var parameter = paramItems.FirstOrDefault(e => e.ParameterName == "screenshot");
-            if (parameter == null)
+            var parameter = paramItems.FirstOrDefault(e => e.ParameterName == "sendImage");
+            if (parameter == null || parameter.ParameterName != "sendImage")
```

Comment:
To be discussed, we can do it in the BaseClientScript in order to create a generic method and check parameters at each start of Execute method.